### DISCUSSION TITLE
Renaming Files / Directories

### DISCUSSION THREAD

**Discussion by jandante** [spatie/file-system-watcher#21] [Ideas]
Lovely package and works like a charm.

Any plans on intercepting renaming files / directories?
For now it triggers a 'fileDeleted' and 'fileCreated'.
I would like to perform other actions on rename then on delete. Or is this already possible?

**Comment by freekmurze** [comment]
If you see a way to achieve this easily, I'd be up for a PR.

I think the underlying js dependency doesn't fire events for those.
